server/storage/backend/tx_buffer.go

```diff
@@ -221,11 +223,13 @@ func (bb *bucketBuffer) Less(i, j int) bool {
 }
 func (bb *bucketBuffer) Swap(i, j int) { bb.buf[i], bb.buf[j] = bb.buf[j], bb.buf[i] }
 
-func (bb *bucketBuffer) Copy() *bucketBuffer {
```

Comment:
Quick thought, should we be retaining the old `Copy()` alongside or are we completely safe to remove it in this pr?

Reply:
hi @jmhbnz 
as this  [comment](https://github.com/etcd-io/etcd/pull/14835#issuecomment-1325387571) says, explicit naming can avoid miss-assumption about copy being equivalent of source.
```CopyUsed``` has only one reference in the etcd code, and the ```bucketBuffer``` is private struct, so i think it's safe.